### ISSUE TITLE
Revisions section incorrect

### ISSUE THREAD

**Issue by galgalesh** [juju/juju-gui#2495]
Url: https://demo.jujucharms.com/?store=cs:cassandra-30

The revision section shows "0 revisions". This is incorrect as you can see on the following page of the jujucharms.com store: https://jujucharms.com/cassandra/30

**Comment by frankban** [comment]
The URL changed (it's now just /cassandra) but the bug is still there. Thanks for reporting!

**Comment by galgalesh** [comment]
I think this should be reopened. The revision section is still not correct.

https://jujucharms.com/cassandra/31

# What it should be

![image](https://cloud.githubusercontent.com/assets/1492981/24198787/935d8d28-0f07-11e7-8594-7a5013fe7962.png)

# What it is

![image](https://cloud.githubusercontent.com/assets/1492981/24198801/a1e892d4-0f07-11e7-99a7-ba30d85ef396.png)

**Comment by frankban** [comment]
We decided to drop this box until we have a good way of displaying that message. The old behavior was confusing as it mixed the charm store revisions (10 revisions) with the vcs commit messages, which we no longer support, as we are no longer ingesting charms from launchpad. Those are two separate concepts, and so we'll restore that box only once we have designed a way to attach a message to the action of uploading a new charm/bundle revision to the charm store.
